english/lunge-for-the-lighter/steep-slope/steep-slope.md

```diff
@@ -5,3 +5,17 @@ this opening before...while being chased by marshmallow men who were shooting
 lasers. As you approach the opening, you see that the slope is much steeper than
 it first appeared, and you can't see where it leads. Just then, you notice a
 big red button above the opening.
+
+You decide against just diving blindly into the opening, and, in a flash of what could be called intuition, you 
+punch the button. This results in two other openings being revealed to the right of the steeply sloping one. You 
+pause. The first of these new openings reveals a tunnel that ascends slightly, and curves gently away to the right.
+Oddly, it seems to made of stone, unlike the rest of what you have seen of this strange place. You can hear the 
+faint sound of dripping liquid coming from the tunnel. It is lit from above, every two meters or so, by small
+lights with wires connecting each of them The second new opening reveals what appears to be a hallway, lit in the 
+same manner as the first, lined with pictures of various types of bird-like lizards wearing very ... um, unique
+clothing. You can just barely hear odd animalistic noises from some point far down the hallway.
+
+[Walk into the stone tunnel](stone-tunnel)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.